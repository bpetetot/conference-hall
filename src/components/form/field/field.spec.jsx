/* eslint-env jest */
import { shallow } from 'enzyme'
import React from 'react'
import Field from './index.js'

describe('components/field', () => {
  it('should render', () => {
    expect(shallow(<Field name="name" label="label" />)).toMatchSnapshot()
  })
})
