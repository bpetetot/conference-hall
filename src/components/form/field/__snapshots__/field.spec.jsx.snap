// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/field should render 1`] = `
<ForwardRef(Field)
  label="label"
  name="name"
  parse={[Function]}
/>
`;
